src/components/ui/Separator/stories/Separator.stories.tsx

```
@@ -34,3 +36,20 @@ export const Color = {
     }
 };
+
+export const Size = () => {
+    return <SandboxEditor>
+        <div className='mt-4 mb-2'>
+            <p className='text-gray-950'>Separator Size</p>
+        </div>
+        <div>
+            <div className='flex flex-col'>
+                {Sizes.map((size, index) => {
+                    return <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)